### ISSUE TITLE
New submissions for Tue,  7 Feb 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#31]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imagingremote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
### Real-Time Image Demoireing on Mobile Devices
 - **Authors:** Yuxin Zhang, Mingbao Lin, Xunchao Li, Han Liu, Guozhi Wang, Fei Chao, Shuai Ren, Yafei Wen, Xiaoxin Chen, Rongrong Ji
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2302.02184
 - **Pdf link:** https://arxiv.org/pdf/2302.02184
 - **Abstract**
 Moire patterns appear frequently when taking photos of digital screens, drastically degrading the image quality. Despite the advance of CNNs in image demoireing, existing networks are with heavy design, causing redundant computation burden for mobile devices. In this paper, we launch the first study on accelerating demoireing networks and propose a dynamic demoireing acceleration method (DDA) towards a real-time deployment on mobile devices. Our stimulus stems from a simple-yet-universal fact that moire patterns often unbalancedly distribute across an image. Consequently, excessive computation is wasted upon non-moire areas. Therefore, we reallocate computation costs in proportion to the complexity of image patches. In order to achieve this aim, we measure the complexity of an image patch by designing a novel moire prior that considers both colorfulness and frequency information of moire patterns. Then, we restore image patches with higher-complexity using larger networks and the ones with lower-complexity are assigned with smaller networks to relieve the computation burden. At last, we train all networks in a parameter-shared supernet paradigm to avoid additional parameter burden. Extensive experiments on several benchmarks demonstrate the efficacy of our proposed DDA. In addition, the acceleration evaluated on the VIVO X80 Pro smartphone equipped with a chip of Snapdragon 8 Gen 1 shows that our method can drastically reduce the inference time, leading to a real-time image demoireing on mobile devices. Source codes and models are released at https://github.com/zyxxmu/DDA
### Distributed Computation Offloading of an application from mobile/IoT  device to cloud
 - **Authors:** Arsalan Liaqat, Saqib Ilyas, Ghazala Mushtaq
 - **Subjects:** Distributed, Parallel, and Cluster Computing (cs.DC)
 - **Arxiv link:** https://arxiv.org/abs/2302.02481
 - **Pdf link:** https://arxiv.org/pdf/2302.02481
 - **Abstract**
 In Covid-19 pandemic, the number of users connecting to the Internet using mobile devices increased. People are doing there every task using mobile phones [16]. These devices are battery-powered and have limited computation capabilities. Their computational capabilities can be enhanced by computation offloading in which required computation is to be done on a third-party server on a cloud instead of the device itself. The cloud offers virtually infinite computation and storage. We proposed that by exploiting parallelism within an application call hierarchy we can decrease the execution time of off-loadable parts and minimize data resend in case of VM crash. We determine function call paths that are independent of each other within an application and schedule each of them on separate VMs in a distributed way. Wherever such independent paths merge, we collapse to a single VM and whenever the paths diverge again, we schedule multiple VMs. If any single VM fails another copy will be created. However, only the code and data associated with the crashed VM needs to be re-transmitted from the client device. In the case of face reorganization application and montage application we decrease execution time to 27.5% and 43.43% respectively. Whereas the data resend in case if any of both VMs crash will be the portion of the application that had been offloaded to respective VM at depending upon the level of parallelism they have which save mobile battery in case of Resend. We will also discuss the energy consumption effect of using multiple Vms for a job VS single Vm for the same job.
### A Quantum Neural Network Regression for Modeling Lithium-ion Battery  Capacity Degradation
 - **Authors:** Anh Phuong Ngo, Nhat Le, Hieu T. Nguyen, Abdullah Eroglu, Duong T. Nguyen
 - **Subjects:** Systems and Control (eess.SY); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2302.02547
 - **Pdf link:** https://arxiv.org/pdf/2302.02547
 - **Abstract**
 Given the high power density low discharge rate and decreasing cost rechargeable lithium-ion batteries LiBs have found a wide range of applications such as power grid level storage systems electric vehicles and mobile devices. Developing a framework to accurately model the nonlinear degradation process of LiBs which is indeed a supervised learning problem becomes an important research topic. This paper presents a classical-quantum hybrid machine learning approach to capture the LiB degradation model that assesses battery cell life loss from operating profiles. Our work is motivated by recent advances in quantum computers as well as the similarity between neural networks and quantum circuits. Similar to adjusting weight parameters in conventional neural networks the parameters of the quantum circuit namely the qubits degree of freedom can be tuned to learn a nonlinear function in a supervised learning fashion. As a proof of concept paper our obtained numerical results with the battery dataset provided by NASA demonstrate the ability of the quantum neural networks in modeling the nonlinear relationship between the degraded capacity and the operating cycles. We also discuss the potential advantage of the quantum approach compared to conventional neural networks in classical computers in dealing with massive data especially in the context of future penetration of EVs and energy storage.
### AuthentiSense: A Scalable Behavioral Biometrics Authentication Scheme  using Few-Shot Learning for Mobile Platforms
 - **Authors:** Hossein Fereidooni, Jan König, Phillip Rieger, Marco Chilese, Bora Gökbakan, Moritz Finke, Alexandra Dmitrienko, Ahmad-Reza Sadeghi
 - **Subjects:** Cryptography and Security (cs.CR)
 - **Arxiv link:** https://arxiv.org/abs/2302.02740
 - **Pdf link:** https://arxiv.org/pdf/2302.02740
 - **Abstract**
 Mobile applications are widely used for online services sharing a large amount of personal data online. One-time authentication techniques such as passwords and physiological biometrics (e.g., fingerprint, face, and iris) have their own advantages but also disadvantages since they can be stolen or emulated, and do not prevent access to the underlying device, once it is unlocked. To address these challenges, complementary authentication systems based on behavioural biometrics have emerged. The goal is to continuously profile users based on their interaction with the mobile device. However, existing behavioural authentication schemes are not (i) user-agnostic meaning that they cannot dynamically handle changes in the user-base without model re-training, or (ii) do not scale well to authenticate millions of users. In this paper, we present AuthentiSense, a user-agnostic, scalable, and efficient behavioural biometrics authentication system that enables continuous authentication and utilizes only motion patterns (i.e., accelerometer, gyroscope and magnetometer data) while users interact with mobile apps. Our approach requires neither manually engineered features nor a significant amount of data for model training. We leverage a few-shot learning technique, called Siamese network, to authenticate users at a large scale. We perform a systematic measurement study and report the impact of the parameters such as interaction time needed for authentication and n-shot verification (comparison with enrollment samples) at the recognition stage. Remarkably, AuthentiSense achieves high accuracy of up to 97% in terms of F1-score even when evaluated in a few-shot fashion that requires only a few behaviour samples per user (3 shots). Our approach accurately authenticates users only after 1 second of user interaction. For AuthentiSense, we report a FAR and FRR of 0.023 and 0.057, respectively.
### Efficient End-to-End Video Question Answering with Pyramidal Multimodal  Transformer
 - **Authors:** Min Peng, Chongyang Wang, Yu Shi, Xiang-Dong Zhou
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2302.02136
 - **Pdf link:** https://arxiv.org/pdf/2302.02136
 - **Abstract**
 This paper presents a new method for end-to-end Video Question Answering (VideoQA), aside from the current popularity of using large-scale pre-training with huge feature extractors. We achieve this with a pyramidal multimodal transformer (PMT) model, which simply incorporates a learnable word embedding layer, a few convolutional and transformer layers. We use the anisotropic pyramid to fulfill video-language interactions across different spatio-temporal scales. In addition to the canonical pyramid, which includes both bottom-up and top-down pathways with lateral connections, novel strategies are proposed to decompose the visual feature stream into spatial and temporal sub-streams at different scales and implement their interactions with the linguistic semantics while preserving the integrity of local and global semantics. We demonstrate better or on-par performances with high computational efficiency against state-of-the-art methods on five VideoQA benchmarks. Our ablation study shows the scalability of our model that achieves competitive results for text-to-video retrieval by leveraging feature extractors with reusable pre-trained weights, and also the effectiveness of the pyramid.
## Keyword: transfer functionmobile
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: volume data
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl2
There is no result 
### Unsupervised Learning for Pilot-free Transmission in 3GPP MIMO Systems
 - **Authors:** Omar M. Sleem, Mohamed Salah Ibrahim, Akshay Malhotra, Mihaela Beluri, Philip Pietraski
 - **Subjects:** Information Theory (cs.IT); Machine Learning (cs.LG); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2302.02191
 - **Pdf link:** https://arxiv.org/pdf/2302.02191
 - **Abstract**
 Reference signals overhead reduction has recently evolved as an effective solution for improving the system spectral efficiency. This paper introduces a new downlink data structure that is free from demodulation reference signals (DM-RS), and hence does not require any channel estimation at the receiver. The new proposed data transmission structure involves a simple repetition step of part of the user data across the different sub-bands. Exploiting the repetition structure at the user side, it is shown that reliable recovery is possible via canonical correlation analysis. This paper also proposes two effective mechanisms for boosting the CCA performance in OFDM systems; one for repetition pattern selection and another to deal with the severe frequency selectivity issues. The proposed approach exhibits favorable complexity-performance tradeoff, rendering it appealing for practical implementation. Numerical results, using a 3GPP link-level testbench, demonstrate the superiority of the proposed approach relative to the state-of-the-art methods.
### Probabilistic-Bits based on Ferroelectric Field-Effect Transistors for  Stochastic Computing
 - **Authors:** Sheng Luo, Yihan He, Baofang Cai, Xiao Gong, Gengchiau Liang
 - **Subjects:** Emerging Technologies (cs.ET)
 - **Arxiv link:** https://arxiv.org/abs/2302.02305
 - **Pdf link:** https://arxiv.org/pdf/2302.02305
 - **Abstract**
 A probabilistic-bit (p-bit) is the fundamental building block in the circuit network of a stochastic computing, and it could produce a continuous random bit-stream with tunable probability. Utilizing the stochasticity in few-domain ferroelectric material(FE), we propose for the first time, the p-bits based on ferroelectric FET. The stochasticity of the FE p-bits stems from the thermal noise-induced lattice vibration, which renders dipole fluctuations and is tunable by an external electric field. The impact of several key FE parameters on p-bits' stochasticity is evaluated, where the domain properties are revealed to play crucial roles. Furthermore, the integer factorization based on FE p-bits circuit network is performed to verify its functionality, and the accuracy is found to depend on FE p-bits' stochasticity. The proposed FE p-bits possess the advantages of both extremely low hardware coast and the compatibility with CMOS-technology, rendering it a promising candidate for stochastic computing applications.
### CIPER: Combining Invariant and Equivariant Representations Using  Contrastive and Predictive Learning
 - **Authors:** Xia Xu, Jochen Triesch
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV); Artificial Intelligence (cs.AI)
 - **Arxiv link:** https://arxiv.org/abs/2302.02330
 - **Pdf link:** https://arxiv.org/pdf/2302.02330
 - **Abstract**
 Self-supervised representation learning (SSRL) methods have shown great success in computer vision. In recent studies, augmentation-based contrastive learning methods have been proposed for learning representations that are invariant or equivariant to pre-defined data augmentation operations. However, invariant or equivariant features favor only specific downstream tasks depending on the augmentations chosen. They may result in poor performance when a downstream task requires the counterpart of those features (e.g., when the task is to recognize hand-written digits while the model learns to be invariant to in-plane image rotations rendering it incapable of distinguishing "9" from "6"). This work introduces Contrastive Invariant and Predictive Equivariant Representation learning (CIPER). CIPER comprises both invariant and equivariant learning objectives using one shared encoder and two different output heads on top of the encoder. One output head is a projection head with a state-of-the-art contrastive objective to encourage invariance to augmentations. The other is a prediction head estimating the augmentation parameters, capturing equivariant features. Both heads are discarded after training and only the encoder is used for downstream tasks. We evaluate our method on static image tasks and time-augmented image datasets. Our results show that CIPER outperforms a baseline contrastive method on various tasks, especially when the downstream task requires the encoding of augmentation-related information.
### When the Ground Truth is not True: Modelling Human Biases in Temporal  Annotations
 - **Authors:** Taku Yamagata, Emma L. Tonkin, Benjamin Arana Sanchez, Ian Craddock, Miquel Perello Nieto, Raul Santos-Rodriguez, Weisong Yang, Peter Flach
 - **Subjects:** Machine Learning (cs.LG); Human-Computer Interaction (cs.HC)
 - **Arxiv link:** https://arxiv.org/abs/2302.02706
 - **Pdf link:** https://arxiv.org/pdf/2302.02706
 - **Abstract**
 In supervised learning, low quality annotations lead to poorly performing classification and detection models, while also rendering evaluation unreliable. This is particularly apparent on temporal data, where annotation quality is affected by multiple factors. For example, in the post-hoc self-reporting of daily activities, cognitive biases are one of the most common ingredients. In particular, reporting the start and duration of an activity after its finalisation may incorporate biases introduced by personal time perceptions, as well as the imprecision and lack of granularity due to time rounding. Here we propose a method to model human biases on temporal annotations and argue for the use of soft labels. Experimental results in synthetic data show that soft labels provide a better approximation of the ground truth for several metrics. We showcase the method on a real dataset of daily activities.
